pkg/expression/builtin_encryption_test.go

```diff
@@ -498,6 +499,25 @@ func TestMD5Hash(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func MD5HashOld(arg string) string {
+	sum := md5.Sum([]byte(arg))
+	return fmt.Sprintf("%x", sum)
+}
+
+func MD5HashNew(arg string) string {
+	sum := md5.Sum([]byte(arg))
+	return hex.EncodeToString(sum[:])
+}
+
+func BenchmarkMD5Hash(b *testing.B) {
+	b.ResetTimer()
+	b.ReportAllocs()
+	for i := 0; i < b.N; i++ {
+		MD5HashOld("abc")
+		//MD5HashNew("abc")
```

Comment:
Can we have both benchmarks? I.e. `BenchmarkMD5HashOld` and `BenchmarkMD5HashNew`.

Reply:
I understand that generally only one benchmark method is used, with two specific methods executed inside, one improved and one old, and then the results are compared using the official tool benchstat

Reply:
> Can we have both benchmarks? I.e. `BenchmarkMD5HashOld` and `BenchmarkMD5HashNew`.

I understand that generally only one benchmark method is used, with two specific methods executed inside, one improved and one old, and then the results are compared using the official tool benchstat